department-of-veterans-affairs/caseflow
# Craig/appeals 13235 cavc dashboard details section

Resolves [APPEALS-13235](https://vajira.max.gov/browse/APPEALS-13235)

### Description
- Added ID and source appeal docket number and decision date to cavc_remand serializer
- Added CavcDashboardDetails component and tests
- Added cavcDashboardSelectors.js file

**Changed the way the remands are passed into the dashboard tab:**
- CavcDashboard passes the remand.id into the tab as a prop instead of the remand itself
- CavcDashboardTab uses newly added selector to get the CavcRemand from state using remand.id and maps that to its props
- This should make it easier to allow the user to make changes by using redux state to set initial component state, then using component state for tracking/sending updates

### Acceptance Criteria
- [x] CAVC details section displays properly

### Testing Plan
1. In demo environment, select a user in the OCC org (OCC_TEAM_USER)
2. Go to the case details page for a CAVC remand case (file number 200000250)
3. From the Case Details page, add '/cavc_dashboard/' to the URL
4. Verify that the dashboard tab contains the CAVC Details section, that it matches the mockups, and that it does not have an Edit button in the top-right of that section
5. Switch user to a user in the OAI org (OAI_TEAM_USER)
6. Go back to the CAVC Dashboard
7. Verify that there is an Edit button visible in the CAVC Details section (behavior of the button will be created in a future story)

### User Facing Changes

<img width="1021" alt="image" src="https://user-images.githubusercontent.com/109101548/217036570-11d56896-34b1-461d-ab82-759990a2ca0d.png">

### Code Documentation Updates
